pinot-segment-local/src/main/java/org/apache/pinot/segment/local/recordtransformer/CompositeTransformer.java

```diff
@@ -87,20 +98,7 @@ public class CompositeTransformer implements RecordTransformer {
    */
   public static List<RecordTransformer> getDefaultTransformers(TableConfig tableConfig, Schema schema) {
```

Comment:
Let's add a TODO to integrate `ComplexTypeTransformer` into the `CompositeTransformer`. We should also document that this method doesn't include enrichers pre complex type transform